java/src/jmri/jmrit/beantable/SignalGroupTableAction.java

```diff
@@ -1277,7 +1277,11 @@ public Object getValueAt(int r, int c) {
             // some error checking
             if (_mastAspectsList == null || r >= aspectList.size()) {
```

Comment:
Thanks for the PR. I think this line also needs to be changed to:

`if (aspectList == null || r >= aspectList.size()) {`